CIScripts/Common/Invoke-CommandWithFunctions.Tests.ps1

```diff
@@ -0,0 +1,126 @@
+Param (
+    [Parameter(Mandatory=$false)] [string] $TestenvConfFile,
+    [Parameter(ValueFromRemainingArguments=$true)] $UnusedParams
+)
+
+. $PSScriptRoot/Init.ps1
+. $PSScriptRoot/../Testenv/Testenv.ps1
+. $PSScriptRoot/../Testenv/Testbed.ps1
+
+. $PSScriptRoot/Invoke-CommandWithFunctions.ps1
+
+Describe "Invoke-CommandWithFunctions tests" -Tags CI, Systest {
+    function Test-CWF {
```

Comment:
What's purpose of this function?

Reply:
To remove "-Session $Session" in every function call.